templates/deploy.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sda.fullname" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "sda.fullname" . }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "sda.fullname" . }}
+    spec:
```

Comment:
For our use case, it's very helpful to be given the chance to insert annotations here, e.g.

```
{{- if .Values.podAnnotations }}
{{- toYaml .Values.podAnnotations | nindent 8 -}}
{{- end }}
```
(or whatever the correct depth is here).

Reply:
https://github.com/neicnordic/sda-helm/blob/cadef3745777dd502bb592c085439c089305d57f/sda-svc/templates/auth-deploy.yaml#L27-L36